internal/resolution/variable_sources/required_package/required_package_test.go

```diff
@@ -15,90 +14,80 @@ import (
 	"github.com/operator-framework/operator-controller/internal/resolution/variable_sources/required_package"
 )
 
-func TestRequiredPackage(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "RequiredPackageVariableSource Suite")
-}
-
-var _ = Describe("RequiredPackageVariable", func() {
+func TestRequiredPackageVariable(t *testing.T) {
+	RegisterTestingT(t)
 	var (
 		rpv            *required_package.RequiredPackageVariable
 		packageName    string
 		bundleEntities []*olmentity.BundleEntity
 	)
-
-	BeforeEach(func() {
-		packageName = "test-package"
-		bundleEntities = []*olmentity.BundleEntity{
-			olmentity.NewBundleEntity(input.NewEntity("bundle-1", map[string]string{
-				property.TypePackage: `{"packageName": "test-package", "version": "1.0.0"}`,
-				property.TypeChannel: `{"channelName":"stable","priority":0}`,
-			})),
-			olmentity.NewBundleEntity(input.NewEntity("bundle-2", map[string]string{
-				property.TypePackage: `{"packageName": "test-package", "version": "2.0.0"}`,
-				property.TypeChannel: `{"channelName":"stable","priority":0}`,
-			})),
-			olmentity.NewBundleEntity(input.NewEntity("bundle-3", map[string]string{
-				property.TypePackage: `{"packageName": "test-package", "version": "3.0.0"}`,
-				property.TypeChannel: `{"channelName":"stable","priority":0}`,
-			})),
-		}
-		rpv = required_package.NewRequiredPackageVariable(packageName, bundleEntities)
-	})
-
-	It("should return the correct package name", func() {
+	packageName = "test-package"
+	bundleEntities = []*olmentity.BundleEntity{
+		olmentity.NewBundleEntity(input.NewEntity("bundle-1", map[string]string{
+			property.TypePackage: `{"packageName": "test-package", "version": "1.0.0"}`,
+			property.TypeChannel: `{"channelName":"stable","priority":0}`,
+		})),
+		olmentity.NewBundleEntity(input.NewEntity("bundle-2", map[string]string{
+			property.TypePackage: `{"packageName": "test-package", "version": "2.0.0"}`,
+			property.TypeChannel: `{"channelName":"stable","priority":0}`,
+		})),
+		olmentity.NewBundleEntity(input.NewEntity("bundle-3", map[string]string{
+			property.TypePackage: `{"packageName": "test-package", "version": "3.0.0"}`,
+			property.TypeChannel: `{"channelName":"stable","priority":0}`,
+		})),
+	}
+	rpv = required_package.NewRequiredPackageVariable(packageName, bundleEntities)
+	t.Run("ShouldReturnCorrectPackageName", func(t *testing.T) {
```

Comment:
I would suggest that you don't do a literal translation of `It()` to `t.Run()`. It's ok to have multiple assertions in a single block of testing code that aren't each in their own t.Run subtest.

Reply:
+1 and if we could use testify for the assertions, even better ^^
@matthew-hagemann thanks so much for this!